src/components/views/rooms/EventTile.tsx

```diff
@@ -1143,13 +1147,15 @@ export default class EventTile extends React.Component<IProps, IState> {
 
                 // tab-index=-1 to allow it to be focusable but do not add tab stop for it, primarily for screen readers
                 return (
-                    React.createElement(this.props.as || "div", {
+                    React.createElement(this.props.as || "li", {
                         "ref": this.ref,
                         "className": classes,
                         "tabIndex": -1,
                         "aria-live": ariaLive,
                         "aria-atomic": "true",
-                        "data-scroll-tokens": this.props["data-scroll-tokens"],
+                        // we can't use local echoes as scroll tokens, because their event IDs change.
+                        // Local echos have a send "status".
+                        "data-scroll-tokens": scrollToken,
```

Comment:
```suggestion
                        "data-scroll-tokens": scrollToken,
```

The comment makes no sense here